src/PokemonGeneral.java

```diff
@@ -0,0 +1,58 @@
+abstract class PokemonGeneral {
```

Comment:
Ik weet niet of "PokemonGeneral" een hele goede naam is voor deze klasse. Ik snap waar het vandaan komt, maar simpelweg "Pokemon" of eventueel "PokemonBase" zou beter zijn. "PokemonGeneral" klinkt als een generieke implementatie van de abstracte super klasse, niet als de abstracte super klasse zelf. Maar goed, dit is aan discussie onderhevig.